lib/grammers-client/src/types/message.rs

```diff
@@ -388,6 +388,50 @@ impl Message {
         self.raw.reply_markup.clone()
     }
 
+    /// Collect all inline callback buttons. Those buttons often appeare in bot chat.
+    /// Buttons could be clicked by [`click_callback_button`]
+    pub fn callback_buttons(&self) -> Option<Vec<tl::types::KeyboardButtonCallback>> {
+        let reply_markup = &self.raw.reply_markup?;
+
+        let mut rtn = Vec::new();
+        if let tl::enums::ReplyMarkup::ReplyInlineMarkup(markup) = reply_markup {
+            for row in markup.rows.iter() {
+                let tl::enums::KeyboardButtonRow::Row(row) = row;
+                for b in row.buttons.iter() {
+                    match b {
+                        tl::enums::KeyboardButton::Callback(callback_b) => {
+                            rtn.push(callback_b.clone());
+                        }
+                        _ => (),
```

Comment:
This means the position in the returned array will not match the one users see in the application. Which would be really confusing.